plugins/woocommerce/changelog/fix-37600

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+The "Update Cart" button does not need an aria-disabled attribute, as it has a disabled attribute, this PR removes it.
```

Comment:
We don't have any specific guidelines around the changelog, but it might be worth re-wording this to reflect the actual change more, you could do something like this:

`Removed aria-disabled attribute from "Update Cart" button as it already has a disabled attribute.`

*editing the changelog file directly to update the description is fine.

Reply:
That makes sense, thanks for the suggestion. Implemented:  https://github.com/woocommerce/woocommerce/pull/37820/commits/e5c306db3d0c99861b301edd62cf326983a53cfd

Reply:
Thanks for updating it, I re-approved! Should be good to merge after all actions finish running.